spec/permissions.json

```diff
@@ -1,66 +1,80 @@
-
 {
   "groups": [
     {
       "name": "super user",
       "id": 0,
       "permissions": {
         "/**": {
-          "permissions": "All"
+          "permissions": "Create | Read | Update | Delete"
         }
```

Comment:
Why this and not a list? This is a list of permissions.

Reply:
The `Access` type can be serialized as any of

- its numeric value,
- the above single-string form, or
- an array of string values.

The reserialization can be any of the above. There's no particular reason I picked the above form, it's just the first form that accepts multiples when you're reading it back. If you prefer the array form it can be that.

Reply:
I think it's better be as close as standard json as possible, especially if  to be able to use external tools (`jq`...)

Reply:
I think it's better to be as close as standard json as possible, especially if  to be able to use external tools (`jq`...)

Reply:
I think it's better to be as close as standard json as possible, especially to be able to use external tools (`jq`...)